test/test.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const parser = require('./../src/parser.js');
+const chai = require('chai');
+const docDefault = fs.readFileSync('./test/config/doc.txt', 'utf8');
+const configDefault = require('./../src/template/configDefault.json');
+const mocha = require('mocha');
+const assert = require('assert');
+
+const main = (testsData) => {
```

Comment:
I think `main` is not the best name for this function

Reply:
@svoboda-rabstvo yes, i changed it. Please, re-check this PR.